dotCMS/src/main/java/com/dotmarketing/factories/DBTreeTransformer.java

```diff
@@ -0,0 +1,52 @@
+package com.dotmarketing.factories;
+
+import com.dotmarketing.beans.Tree;
+
+import java.util.List;
+import java.util.Map;
+import java.util.stream.Collectors;
+
+import com.google.common.collect.Lists;
+
+
+public class DBTreeTransformer{
+    
+    private final List<Tree> trees;
+    
+    public DBTreeTransformer (List<Map<String, Object>>list ){
+        this.trees =  list
+        .stream()
+        .map(row -> toTree(row))
+        .collect(Collectors.toList());
+
+    }
+    
+    public Tree tree() {
+        return trees.stream().findFirst().orElse(null);
+       
+    }
+    public List<Tree> trees() {
+        return trees;
```

Comment:
![MINOR](https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR') Return a copy of "trees". [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=squid%3AS2384)